src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/test_batch_mgmt_commands.py

```diff
@@ -20,187 +20,166 @@ class BatchMgmtScenarioTests(ScenarioTest):  # pylint: disable=too-many-instance
 
     @ResourceGroupPreparer(location='northeurope')
     def test_batch_account_cmd(self, resource_group):
+        self.skipTest("")
```

Comment:
Why the skip?

Reply:
This is the test that uses keyvault - would you prefer I just let it fail until keyvault is finished?

Reply:
Yeah, that's probably best.  In the issue for the batch module, just annotate it as a known failing test so we can get it fixed once KeyVault is unblocked.